test/es-module/test-esm-import-meta.mjs

```diff
@@ -18,3 +18,13 @@ for (const descriptor of Object.values(descriptors)) {
 
 const urlReg = /^file:\/\/\/.*\/test\/es-module\/test-esm-import-meta\.mjs$/;
 assert(import.meta.url.match(urlReg));
+
+const dirReg = /^\/.*\/test\/es-module$/;
+assert.match(import.meta.dirname, dirReg);
+
+const fileReg = /^\/.*\/test\/es-module\/test-esm-import-meta\.mjs$/;
+assert.match(import.meta.filename, fileReg);
```

Comment:
Do we need any separate tests for Windows paths? What about `file:` URLs that are loading network paths, like `file://server/folder/file.js`?

Reply:
> Do we need any separate tests for Windows paths?

Is this done for the CJS implementation? What would we actually be testing? That `path.dirname` works with Windows paths?

> What about `file:` URLs that are loading network paths, like `file://server/folder/file.js`?

That looks like a mounted file system to me.